command/git/commands.go

```diff
@@ -0,0 +1,145 @@
+package git
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/command"
+)
+
+// Init creates an empty Git reporitory or reinitializes an existing one.
+func (g *Git) Init() *command.Model {
+	return g.Command("init")
+}
+
+// Clone a repository into a new directory
+func (g *Git) Clone(repo string) *command.Model {
+	return g.Command("clone", repo)
+}
+
+// RemoteList shows a list of existing remote urls with remote names
+func (g *Git) RemoteList() *command.Model {
+	return g.Command("remote", "-v")
+}
+
+// RemoteAdd adds a remote named <name> for the repository at <url>.
+func (g *Git) RemoteAdd(name, url string) *command.Model {
+	return g.Command("remote", "add", name, url)
+}
+
+// Fetch downloads objects and refs from another repository.
+func (g *Git) Fetch(depth int, opts ...string) *command.Model {
+	args := []string{"fetch"}
+	if depth != 0 {
+		args = append(args, "--depth="+strconv.Itoa(depth))
+	}
+	args = append(args, opts...)
+	return g.Command(args...)
+}
+
+// FetchPR downloads objects and refs for the pull request branch.
+func (g *Git) FetchPR(branch string) *command.Model {
```

Comment:
do we need this in the lib? if yes it is only for github prs, right? if yes rename it!